src/components/payment/fee-summary.vue

```diff
@@ -1,39 +1,61 @@
 <template>
   <div class="fee-summary">
-    <v-row align="center" justify="center">
+    <v-row no-gutters align="center" justify="center">
       <v-col cols="10">
-        <div class="font-weight-bold px-3 pb-3 header">Payment Details</div>
-
-        <v-alert v-if="fetchError" color="error" icon="mdi-alert" outlined class="my-0" v-html="fetchError" />
-
-        <v-slide-y-transition group tag="ul" class="fee-list py-2" v-show="!fetchError">
-          <template
-            v-for="lineItem in fees"
-            v-show="(totalFees > 0 && lineItem.filingFees) || (totalFees === 0)"
-            >
-            <li class="container fee-list__item"
-              :key="lineItem.filingType"
-              >
-              <div class="fee-list__item-name">{{lineItem.filingType}}</div>
-              <div class="fee-list__item-value" v-if="lineItem.filingFees > 0">${{lineItem.filingFees.toFixed(2)}}</div>
-              <div class="fee-list__item-value" v-else>No Fee</div>
-            </li>
-            <li class="container fee-list__item"
-              v-if="lineItem.priorityFees"
-              :key="lineItem.filingTypeCode+'-priority'"
-              >
-              <div class="fee-list__item-name">Priority Request fee</div>
-              <div class="fee-list__item-value">${{lineItem.priorityFees.toFixed(2)}}</div>
-            </li>
-            <li class="container fee-list__item"
-              v-if="lineItem.serviceFees"
-              :key="lineItem.filingTypeCode+'-transaction'"
-              >
-              <div class="fee-list__item-name">Service fee</div>
-              <div class="fee-list__item-value">${{lineItem.serviceFees.toFixed(2)}}</div>
-            </li>
-          </template>
-        </v-slide-y-transition>
+        <v-row no-gutters class="pt-8 pb-4 header">
```

Comment:
Why use a v-row here with no v-col in it? Would a div work as well?

Reply:
updated - this one, checking to see if I can change the ones cam questioned below

Reply:
Smells clean now!

(It was a code smell to use v-rows without v-cols, haha.)